Helper/Data.php

```diff
@@ -1506,6 +1513,47 @@ public function initializeAdyenClient($storeId = null, $apiKey = null)
         return $client;
     }
 
+    /**
+     * Initializes and returns Adyen Client and sets the required parameters of it
+     *
+     * @param null|int|string $storeId
+     * @param string|null $apiKey
+     * @return \Adyen\Client
+     * @throws \Adyen\AdyenException
+     */
+    public function initializeAdyenMotoClient($motoMerchantAccount, $storeId = null)
```

Comment:
It could be nice to add an option parameter to `initializeAdyenClient()` method as `motoMerchantAccount` and discard the new method.